src/tracing/errors.ts

```diff
@@ -40,6 +40,21 @@ export const filterKnownErrors: Required<ClientOptions>['beforeSend'] = (event:
     // If the error is based on a user rejecting, it should not be considered an exception.
     if (didUserReject(error)) return null
 
+    /*
+     * This ignores 499 errors, which are caused by Cloudflare when a request is cancelled.
+     * CF claims that some number of these is expected, and that they should be ignored.
+     */
+    if (error.message.match(/Loading chunk \d+ failed\. \(error: .+\.chunk\.js\)/)) {
+      const asset = error.message.match(/https?:\/\/.+?\.chunk\.js/)?.[0]
+      const resource = [...(performance?.getEntriesByType('resource') ?? [])].find(({ name }) => name === asset)
+      // `responseStatus` is not on the `ResponseStatus` type, because it's only supported on some browsers at the moment.
+      const status = (resource as any)?.responseStatus
+      // If the status if 499, then we ignore. If there's no status, for now we'll ignore all chunk loading errors also.
```

Comment:
Why are we ignoring unknown errors? Why not _just_ 499s?

Reply:
Also related to my comment https://github.com/Uniswap/interface/pull/6365#discussion_r1172885316

Reply:
we're ignoring them bc responseStatus is only supported on Chrome. otherwise we would be logging ChunkErrors from other browsers that are likely just 499s.